server-common/src/main/java/org/apache/gravitino/server/authentication/KerberosServerUtils.java

```diff
@@ -248,22 +249,22 @@ public static String getTokenServerName(byte[] rawToken) {
       throw new IllegalArgumentException("Malformed gss token");
     }
     // InnerContextToken ::= {
-    //     token-id[1]
-    //     AP-REQ
+    // token-id[1]
+    // AP-REQ
     // }
     if (token.next().getTag() != 1) {
       throw new IllegalArgumentException("Not an AP-REQ token");
     }
     // AP-REQ ::= [APPLICATION 14] SEQUENCE {
-    //     ticket[3]      Ticket
+    // ticket[3] Ticket
     // }
     DER ticket = token.next().get(0x6e, 0x30, 0xa3, 0x61, 0x30);
     // Ticket ::= [APPLICATION 1] SEQUENCE {
-    //     realm[1]       String
-    //     sname[2]       PrincipalName
+    // realm[1] String
+    // sname[2] PrincipalName
     // }
     // PrincipalName ::= SEQUENCE {
-    //     name-string[1] SEQUENCE OF String
+    // name-string[1] SEQUENCE OF String
```

Comment:
I think the changes here and above are not necessary, it would be better not to change them.